docs/content/python/guides/roles/getting-started.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Getting started
+weight: 2
+description: >
+    Get started with Oso Roles
+---
+
+# Getting started
+
+When managing access to resources within an application, it can be useful to
+group permissions into **roles**, and assign these roles to users. This is
+known as **Role-Based Access Control (RBAC).** The `Oso` core library
+comes with built in configuration for role-based access control.
+
+In this guide, we'll walk through the basics of starting to use the
+roles feature.
+
+## Setting up the Oso Instance
+
+First, we'll cover some of the basics of integrating Oso into your
+application.
+
+The `oso.Oso` class is the entrypoint to using Oso in our application.
+We usually will have a global instance that is created
+during application initialization and shared across requests. In a Flask
+application, you may attach this instance to the global flask object, or the
+current application if it needs to be accessed outside of the application
+initialization process.
+
+### Loading our policy
+
+Oso uses the Polar language to define authorization policies. An
+authorization policy specifies what requests are allowed and what data a
+user can access. The policy is stored in a Polar file, along with your
+code.
+
+Load the policy with the `Oso.load_file` function.
+
+```py
+oso.load_file("authorization.polar")
+```
+
+### Enable Oso roles
+
+
+In order to enable the built-in roles features, we 
+`oso.Oso.enable_roles` method:
+
+```py
+oso.enable_roles()
+```
+
+{{% callout "Load policies before enabling roles" "blue" %}}
+
+Oso will validate your roles configuration when you call `enable_roles`.
+You must load all policy files before enabling roles.
+
+{{% /callout %}}
+
+
+## Controlling access with roles
+
+Now, let's write our first rules that use role based access control. To
+setup the role library, we must:
+
+1. Add role and resource configurations to our policy.
+2. Use the `role_allows` method in our policy.
+3. Assign roles to users.
+
+### Configuring our first resource
+
+Roles in Oso are scoped to resources. A role is a grouping of
+permissions that may be performed on that resource. Roles are assigned
```

Comment:
we can workshop this more later, but I think permission == (action, resource)

```suggestion
actions that may be performed on that resource. Roles are assigned
```

ah... which I guess is probably confusing given that the key in the `roles` dict is named "permissions"... although I think there's an implicit resource there (either the resource on which the role is defined or a related resource for namespaced actions).

I guess I'm fine with "a role is a group of permissions", but I still don't like "permissions that may be performed on that resource"